common/test/acceptance/fixtures/__init__.py

```diff
@@ -1,11 +1,15 @@
 import os
 
 
+HOSTNAME = os.environ.get('BOK_CHOY_HOSTNAME', 'localhost')
+CMS_PORT = os.environ.get('BOK_CHOY_CMS_PORT', '8031')
```

Comment:
Ah, ok.  8031 is the fallback used by Vagrant devstack, where an environment variable isn't currently set to specify the port.  It's currently hardcoded in about a dozen places, because a lot of test files and fixtures use it without referencing the Django settings.  I don't think this PR makes that any worse, but it doesn't try to fix it either.  That seems like it deserves a separate ticket.